docs/configuration.md

```diff
@@ -1,492 +1,442 @@
+In a typical setup where you run your Registry from the official image, you can
+specify a configuration variable from the environment by passing `-e` arguments
+to your `docker run` stanza or from within a Dockerfile using the `ENV`
+instruction.
 
 To override a configuration option, create an environment variable named
-`REGISTRY_variable` where *`variable`* is the name of the configuration option
+`REGISTRY_variable` where `variable` is the name of the configuration option
 and the `_` (underscore) represents indention levels. For example, you can
 configure the `rootdirectory` of the `filesystem` storage backend:
 
-    storage:
-      filesystem:
-        rootdirectory: /var/lib/registry
+```none
+storage:
+  filesystem:
+    rootdirectory: /var/lib/registry
+```
 
 To override this value, set an environment variable like this:
 
-    REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY=/somewhere
+```none
+REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY=/somewhere
+```
 
 This variable overrides the `/var/lib/registry` value to the `/somewhere`
 directory.
 
->**NOTE**: It is highly recommended to create a base configuration file with which environment variables can be used to tweak individual values.  Overriding configuration sections with environment variables is not recommended.
+> **Note**: Create a base configuration file with environment variables that can
+> be configured to tweak individual values. Overriding configuration sections
+> with environment variables is not recommended.
 
 ## Overriding the entire configuration file
 
-If the default configuration is not a sound basis for your usage, or if you are having issues overriding keys from the environment, you can specify an alternate YAML configuration file by mounting it as a volume in the container.
+If the default configuration is not a sound basis for your usage, or if you are
+having issues overriding keys from the environment, you can specify an alternate
+YAML configuration file by mounting it as a volume in the container.
 
-Typically, create a new configuration file from scratch, and call it `config.yml`, then:
+Typically, create a new configuration file from scratch,named `config.yml`, then
+specify it in the `docker run` command:
 
-    docker run -d -p 5000:5000 --restart=always --name registry \
-      -v `pwd`/config.yml:/etc/docker/registry/config.yml \
-      registry:2
+```bash
+$ docker run -d -p 5000:5000 --restart=always --name registry \
+             -v `pwd`/config.yml:/etc/docker/registry/config.yml \
+             registry:2
+```
 
-You can (and probably should) use [this as a starting point](https://github.com/docker/distribution/blob/master/cmd/registry/config-example.yml).
+Use this
+[example YAML file](https://github.com/docker/distribution/blob/master/cmd/registry/config-example.yml)
```

Comment:
This should link to the 2.6 branch/tag in case the "master" version is not compatible with 2.6